TestApp/DeviceAgentUnitTests/Utilities/SpringboardAlertsTest.m

```diff
@@ -33,6 +37,65 @@ - (void)testAlerts {
     expect(actual.count > 0).to.beTruthy();
 }
 
+- (BOOL)containsLnaguage:(NSString*)preferredLanguage:(SpringBoardAlerts*)obj {
+    BOOL resEn = NO;
+    BOOL resPreferred = NO;
+    NSString *englishAlert = @"No SIM Card Installed";
+    for(SpringBoardAlert *alert in obj.alerts){
+        if([alert matchesAlertTitle: preferredLanguage]){
+            resPreferred = YES;
+        }
+        if([alert matchesAlertTitle: englishAlert]){
+            resEn = YES;
+        }
+    }
+    return resEn && resPreferred;
+}
+
+- (void)testLanguage {
+    NSArray<NSString *> * arrayLanguages = [NSArray arrayWithObjects:@"data available to nearby bluetooth devices", @"souhaite accéder à vos mouvements et vos activités physiques", @"Deseja Ter Acesso às Suas Fotos", @"доступ к Вашим геоданным",nil];
+    BOOL resFinal = NO;
+    id localeMock = OCMClassMock([NSLocale class]);
+    NSArray<NSString *> * array = [NSArray arrayWithObject:@"fr-Pytin"];
+    OCMStub([localeMock preferredLanguages]).andReturn(array);
+    SpringBoardAlerts *obj = [[SpringBoardAlerts alloc] init_private];
+    
+    expect(obj).toNot.equal(NULL);
+    
+    localeMock = OCMClassMock([NSLocale class]);
+    OCMStub([localeMock preferredLanguages]).andReturn(array);
+    obj = [[SpringBoardAlerts alloc] init_private];
+    resFinal = [self containsLnaguage:arrayLanguages[1]:obj];
```

Comment:
We should get rid of `arrayLanguages` because it is very unclear.
I suggest to rework function `containsLanguage` just to check if array contains alert or not.
Then we will have to write the following tests:
```
obj = [[SpringBoardAlerts alloc] init_private];
expect([self containsLanguage:@"French phrase"]).to.beTruthy()
expect([self containsLanguage:@"English phrase"]).to.beFalsy()
expect([self containsLanguage:@"Russian phrase"]).to.beFalsy()
```